### PR TITLE
Fix ctags bug by resetting protobuf syntax

### PR DESCRIPTION
**Pull request by chrismwendt**
Includes https://github.com/universal-ctags/ctags/pull/3300

## Test plan

Ran locally and verified it parsed correctly.

### CODE REVIEW

File: cmd/symbols/ctags-install-alpine.sh [hunk 1]
```diff
@@ -4,7 +4,9 @@
 
 # Commit hash of github.com/universal-ctags/ctags.
 # Last bumped 2022-02-10
```
Comment (efritz) [inline cmd/symbols/ctags-install-alpine.sh:6]: Need to update this date?

**Review by efritz — APPROVED** [review]

**Comment by efritz** [comment]
Does this affect the indexed symbols search path? Do we need to update a dependency in zoekt as well?

**Comment by chrismwendt** [comment]
I think Zoekt uses whatever `ctags` is installed in the environment. This previous PR updated sourcegraph/go-tags as well, but that package uses the environment's `ctags`:

https://github.com/sourcegraph/sourcegraph/pull/30946/files?w=1

**Comment by keegancsmith** [comment]
You do need to update zoekt as well https://github.com/sourcegraph/zoekt/blob/master/Dockerfile#L21-L24

Done here https://github.com/sourcegraph/zoekt/commit/e68ec97a484058ce602f4cf0ce5ba15f2ad9532b

**Pull request merged**